# Goal
geometry_msgs/Pose target_pose
string target_map
---
# Result  
bool success
string message
---
# Feedback
string current_status